src/utils/accessibility/accessibility.js

```diff
@@ -3,20 +3,32 @@ function isLargeScale(fontSize, isFontBold) {
   return isFontBold ? (fontSize >= 14) : (fontSize >= 18);
 }
 
-function accessibleContrast(accessibilityLevel, fontSize, isFontBold) {
+function fontSizeInPt(fontSize,fontUnitOfMeasure) {
```

Comment:
I hope to find time to integrate [Prettier](https://github.com/prettier/prettier) which auto formats the code, so we don't discuss styling issues in the future.